pages/information/estate.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: category
+title: Estimating Estate Emissions
+permalink: /information/estate
```

Comment:
```suggestion
permalink: /information/estate
published: false
```
Add this to prevent this page from being published, just needs it removed from the information.yml as well